code/src/java/pcgen/io/exporttoken/WeaponToken.java

```diff
@@ -1887,8 +1887,8 @@ else if (hitMode == HITMODE_OHHIT)
 				baseBonus += pc.getOffHandLightBonus();
 			}
 
-			if ((hitMode == HITMODE_TWOHIT) || (hitMode == HITMODE_OHHIT) || (hitMode == HITMODE_TWFOHL)
-				|| (hitMode == HITMODE_TWFOHH))
+			if ((hitMode == HITMODE_OHHIT) || (hitMode == HITMODE_TWFOHL)
+					|| (hitMode == HITMODE_TWFOHH))
```

Comment:
Doesn't this strike you more as a bug than a potential optimization?

Reply:
I don't see the duplicate either?

Reply:
I don't think its a bug, but you're right that this is weird. better fix applied.